interop-testing/src/test/java/io/grpc/testing/integration/TestLogHandler.java

```diff
@@ -0,0 +1,38 @@
+package io.grpc.testing.integration;
+
+import java.text.MessageFormat;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.logging.Handler;
+import java.util.logging.LogRecord;
+
+public class TestLogHandler extends Handler {
```

Comment:
Can this just be made an inner class?  If it is only used in one place, I would inline it in said class.